From 156df136a053e509e78abeb0e503ff3020f12fe0 Mon Sep 17 00:00:00 2001
From: "Benjamin A. Beasley" <code@musicinmybrain.net>
Date: Thu, 22 Aug 2024 11:48:36 -0400
Subject: [PATCH 1/3] Downstream-only: Omit doctests that require hifitime

It is not worth packaging it solely for a couple of tiny examples.
---
 COMPARE.md | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/COMPARE.md b/COMPARE.md
index b461ba5..d10072d 100644
--- a/COMPARE.md
+++ b/COMPARE.md
@@ -1148,7 +1148,7 @@ report the accurate duration. With that said, to get an accurate duration, one
 of the two time scales has to be converted to to TAI (or another time scale
 which does not have leap seconds):
 
-```rust
+```rust,ignore
 use hifitime::{Duration, Epoch, TimeScale};
 
 fn main() -> anyhow::Result<()> {
@@ -1204,7 +1204,7 @@ In contrast, `hifitime` appears to use saturating arithmetic everywhere (I've
 not been able to find this behavior documented though, so I'm not clear on what
 the intended semantics are):
 
-```rust
+```rust,ignore
 use hifitime::{Duration, Epoch};
 
 fn main() -> anyhow::Result<()> {
-- 
2.54.0

